website/source/docs/enterprise/vcs/bitbucket.html.md

```diff
@@ -72,14 +72,16 @@ three pieces of information:
 format _username/repository_.
 
 - **Bitbucket Cloud branch** - The branch from which to ingress new versions. This
-defaults to the value GitHub provides as the default branch for this repository.
+defaults to the value Bitbucket Cloud provides as the default branch for this repository.
 
 - **Path to directory of Terraform files** - The repository's subdirectory that
 contains its terraform files. This defaults to the root of the repository.
 
+**Note**: Users creating, updating, or deleting webhooks via the API must have `owner` or `admin` permissions enabled on the target Bitbucket Cloud repository. To update user permissions on the target repository the repository owner can visit: https://bitbucket.org/your-username/your-repository/admin/access
```

Comment:
going to forego the markdown in this case as the link is more of an example than a link that will actually take you anywhere. if i were to say something like go here. i'd display here in MD like: [here](https://example.com)